roles/systemd_resolved/tasks/resolv.conf.yml

```diff
@@ -1,8 +1,18 @@
 ---
-- name: Ensure resolv.conf
+- name: Ensure resolv.conf (classic)
+  ansible.builtin.file:
+    src: /run/systemd/resolve/resolv.conf
+    dest: /etc/resolv.conf
+    state: link
+    force: true
+  become: true
+  when: ( systemd_resolved_dnsstublistener | regex_search("no|false|0", ignorecase=True) )
```

Comment:
please don't.
let's keep things simple.

there has to be a better way to do that beside using regex here
something like 'when: not systemd_resolved_dnsstublistener' ?

Reply:
These are the possible Options you can find in resolved.conf 

```
DNSStubListener=
Takes a boolean argument or one of "udp" and "tcp". If "udp", a DNS stub resolver will listen for UDP requests on addresses 127.0.0.53 and 127.0.0.54, port 53. If "tcp", the stub will listen for TCP requests on the same addresses and port. If "yes" (the default), the stub listens for both UDP and TCP requests. If "no", the stub listener is disabled.
```

[Man](https://www.freedesktop.org/software/systemd/man/latest/resolved.conf.html)